extensions/wikia/Email/i18n/Welcome.i18n.php

```diff
@@ -0,0 +1,33 @@
+<?php
+$messages = [];
+
+$messages['en'] = array(
+	'emailext-welcome-subject' => "Welcome to Wikia, $1!",
+	'emailext-welcome-summary' => "Congrats, you’re a Wikian! You are now part of a universe created entirely for fans and authored by millions of other fans, just like you. <br><br>So what’s next?",
```

Comment:
I don't think we should have HTML in these messages. If it needs to go on another line we should probably make it a separate message.

As part of SOC-532 I added another template var to greeting.mustache called "extended-summary", this would probably be a good use for that. See [here](https://github.com/Wikia/app/blob/SOC-532/extensions/wikia/Email/templates/greeting.mustache#L4-L6). Unfortunately it's part of a massive commit, but you could probably just copy and paste that extra bit into greeting.mustache. As long as it's the same I don't think there'll be conflicts.